frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -126,6 +127,16 @@ const WARNING_TYPE_RENDERER = {
             </>
         )
     },
+    message_size_too_large: function Render(warning: IngestionWarning): JSX.Element {
+        const details = warning.details as {
+            eventUuid: string
+        }
+        return (
+            <>
+                Discarded event that exceeded 1MB in size after processing: <code>{details.eventUuid}</code>
+            </>
```

Comment:
I don't love that there isn't much useful detail to show here. It'd be really helpful to attribute _why_ the event got so large (was it persons denormalization? something a plugin/app did? _too many_ plugins? something else?), but I think that's going to be challenging to derive. "1MB" also feels like an implementation detail leak to me but I figured it at least gives the user _something_ to contextualize the size with rather than just a vague description of the event being "too big."

Reply:
We could add `distinct_id` here too, which might give users a better idea for where the event came from or what the problem likely is.

Reply:
Updated with [cecd9eb](https://github.com/PostHog/posthog/pull/18318/commits/cecd9eb19e49f5feb3d618c7f786ec52e5258532):

![localhost_8000_data-management_ingestion-warnings (2)](https://github.com/PostHog/posthog/assets/65315/140c8277-143c-49f0-989d-c5bbefe39c94)